### ISSUE TITLE
Is it possible to attach screenshot to the playwright html report

### ISSUE THREAD

**Issue by HanishaRampe** [vitalets/playwright-bdd#38]

**Comment by vitalets** [comment]
Yes. There is nothing special with `playwright-bdd` here. Just set config option [`screenshot`](https://playwright.dev/docs/api/class-testoptions#test-options-screenshot) to `on | only-on-failure`.
